wagtail/wagtailcore/blocks/field_block.py

```diff
@@ -139,6 +139,24 @@ class Meta:
         icon = "pilcrow"
 
 
+class BlockQuoteBlock(TextBlock):
+
+    def __init__(self, required=True, help_text=None, max_length=None, min_length=None, **kwargs):
+        super(BlockQuoteBlock, self).__init__(**kwargs)
```

Comment:
Are the `required` / `help_text` / `max_length` / `min_length` parameters respected here? It looks like those are just thrown away in this definition.

You can probably just omit `__init__` and `get_searchable_content` here, because they aren't doing anything that `TextBlock` isn't doing already.